Buttons

moonclash opened the pull request.

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into buttons

Commit:
Added two buttons concepts

Added two buttons concepts about an <a> button and a HTML5 <button>
element.

moonclash commented:
# **Added two button concepts**

![buttons](https://cloud.githubusercontent.com/assets/12881226/14233317/db672ac2-f9bc-11e5-96e1-f7b62d951fce.png)

Commit:
Removed whitespace

Commit:
Removed border-radius transition

william-abboud commented on src/index.html:
```diff
@@ -28,6 +28,12 @@
+  <div class="buttons-container">
+    <a href="#" class="btn">get started</a>
+
+    <button class="form-btn">join the ranks</button>
```
Okay so before you make a form button just make a primary button. So these styles you've applied at the moment to `.form-btn` should be generic for say a `primary-btn`.
A `button` element should inherit the styles for the primary btn by default without having to add a class. So do something like this:

``` css
button, .primary-btn {
  // Styles for a default button here
}
```

``` html
<button> Register </button>
<a class="primary-btn" href="#"> Enter </a>
```

This is the final effect we are looking for ^

william-abboud commented on src/styles/Base/default.scss:
```diff
@@ -14,5 +14,5 @@ ol {
 a {
   text-decoration: none;
-  color: $black;
+  color: $black-500;
```
Good spot !

william-abboud commented on src/styles/Elements/buttons.scss:
```diff
@@ -0,0 +1,33 @@
+  border-radius: 18px;
+  box-shadow: 1px 1px 2px $black-400;
+  display: inline-block;
+  padding: 7px 13px;
```
Use `rem` units everywhere except for `border-radius`, `border`, `box-shadow` :)
Also add variation for `large` `small` `medium` buttons aka different paddings ;)

P.S `1rem` = `10px`

william-abboud commented on src/styles/Elements/buttons.scss:
```diff
@@ -0,0 +1,33 @@
+  transform: scale(1);
+  transition: 0.1s all ease-in-out;
+
+
```
Remove this empty line one is enough :)

william-abboud commented on src/styles/Elements/buttons.scss:
```diff
@@ -0,0 +1,33 @@
+  margin-left: 50px;
+  padding: 7px 13px;
+  transition: 0.1s all ease-in-out;
+  &:hover {
```
Separate &:hover with a line break

Commit:
Added several changes to buttons

Added default stylings for buttons, changed size unites to rems, added
stylings for small,medium and large size buttons.
